src/app/asset-page/asset-page.component.ts

```diff
@@ -870,6 +866,42 @@ export class AssetPage implements OnInit, OnDestroy {
         return false
     }
 
+    /**
+     * trackItemView
+     * Builds and sends captain's log event for 'artstor_item_view'
+     * @param asset
+     * @returns void
+     */
+
+    private trackItemView(asset: Asset): void {
+
+      let reasonForAuth: string[]
+      let hasAccess: boolean = true
+
+      if (this.showAccessDeniedModal) {
+        reasonForAuth = ['not_authorized']
+        hasAccess = false
+      }
+      else if (this._auth.isPublicOnly()) {
```

Comment:
Mind putting the `else if` on the same line as the closing bracket for the `if`?

Reply:
@crystalfinch Good eye, thanks! Updated.